skins/laika/src/components/DonationCommentPopUp.vue

```diff
@@ -69,6 +69,9 @@ export default Vue.extend( {
 			},
 		},
 	},
+	created: function () {
```

Comment:
IMHO this  should be `mounted` instead of `created`, because the DOM nodes don't exist at this point. See https://alligator.io/vuejs/component-lifecycle/